### ISSUE TITLE
Structure pattern: the Home link in the breadcrumb lacks any title or text

### ISSUE THREAD

**Issue by erral** [plone/mockup#955]
To make the breadcrumb in the pattern structure accessible, the home link should have a title.

@ionlizarazu
